edb/pgsql/compiler/expr.py

```diff
@@ -433,6 +433,10 @@ def compile_operator(
     lexpr = rexpr = None
     result: Optional[pgast.BaseExpr] = None
 
+    if expr.body is not None:
+        raise NotImplementedError(
+            "support for generic inline operator calls is not implemented")
```

Comment:
Maybe move body from `irast.Call` to `irast.FunctionCall`?

Reply:
OK

Reply:
I feel like inline functions ought to be a separate class entirely.

Reply:
OK, I think my over-ambitious naming of the PR got people spooked. This isn't really anywhere close to actual inline functions.  Instead, think of this as support for special functions that only need to be "special" in IR, but don't require special SQL handling.

We'll revisit this when we start adding _actual_ inline functions.